metabase/metabase
# Ellipsify long dashboard filter values (strings only)

Closes #46189

Sets a max width for string filter widgets so they don't go off-screen if a value is long enough

### To verify

1. Open a dashboard with at least one question
2. Add a string filter (don't forget to link it to a column so the filter is visible after leaving editing mode)
3. When editing the string filter, set pick "Input box" in the "How should people filter on this column?" section of the filter sidebar
4. Save the dashboard
5. Add a long enough value (could be a few sentences)
6. Ensure the filter component doesn't go off-screen and that it ellipsifies its contents. Ensure you can hover the filter and see the full value in a tooltip
7. Ensure filters look as expected with shorter values

### Demo

**Before**

![before](https://github.com/user-attachments/assets/c0f65717-ed36-4087-bc1b-160a7ca1e163)

**After**

![after](https://github.com/user-attachments/assets/ec934789-4062-4143-ad2b-4a44eaa01203)
